docs/video-miners/getting-started/testing/testnet.md

```diff
@@ -3,38 +3,40 @@ title: Running on Testnet
+This guide will walk you through the process of running an orchestrator on the Arbitrum Rinkeby testnet. This is an important step to (1) ensure that your orchestrator is correctly configured, and (2) take any troubleshooting steps and make any tweaks in a lower-risk environment than mainnet.
 
-1. **Acquire some Rinkeby ETH from a faucet**
+1. **Acquire some Rinkeby ETH from a faucet and bridge it to Arbitrum**
 
-You can use an existing third-party faucet for this. You will want to send the ETH to the wallet that you will be using to run your orchestrator. We recommend [the official Rinkeby faucet](https://www.rinkeby.io/#faucet) (requires Twitter or Facebook auth), or [the Chainlink faucet](https://faucets.chain.link/rinkeby) (no auth required). Most faucets provide ~0.1 ETH per request, but you can use them multiple times if you need more.
+You can use an existing third-party faucet for this. We recommend [the official Rinkeby faucet](https://www.rinkeby.io/#faucet) (requires Twitter or Facebook auth), or [the Chainlink faucet](https://faucets.chain.link/rinkeby) (no auth required). Most faucets provide ~0.1 ETH per request, but you can use them multiple times if you need more.
 
-2. **Get a Rinkeby RPC url, either by running a node or by using a url from Infura or Alchemy**
+Once you've acquired the Rinkeby ETH, use the [Arbitrum bridge](https://bridge.arbitrum.io) to send Rinkeby ETH to Arbitrum. After bridging, you will want to send the Arbitrum Rinkeby ETH to the wallet that you will be using to run your orchestrator.
 
-To run your own `geth` node, please follow [Rinkeby’s official guide](https://www.rinkeby.io/#geth). If you’d prefer to use an [Infura](https://infura.io/) or [Alchemy](https://www.alchemy.com/) RPC url, please follow the guides on their respective sites.
+2. **Get an Arbitrum Rinkeby RPC url, either by running a node or by using a url from Infura or Alchemy**
 
-3. **Run the Livepeer CLI, pointing at Rinkeby**
+We recommend using [the Offchain Labs public testnet endpoint](https://developer.offchainlabs.com/docs/public_testnet). Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/docs/running_node) or acquire an Arbitrum RPC url using a third-party service like [Alchemy](https://www.alchemy.com/) or [Infura](https://infura.io/). 
+
+3. **Run the Livepeer CLI, pointing at Arbitrum Rinkeby**
 
 If you have not already installed the Livepeer CLI, please refer to [this guide](https://livepeer.org/docs/installation/install-livepeer/overview) before running the command below
 
 ```bash
 $: livepeer \
-    -network rinkeby
-    -ethUrl <ETH_URL> # RPC Url for Rinkeby provider, acquired in Step 2 above
+    -network arbitrum-one-rinkeby
+    -ethUrl <ETH_URL> # RPC Url for Arbitrum Rinkeby provider, acquired in Step 2 above
```

Comment:
Consider adding here the official endpoint `https://rinkeby.arbitrum.io/rpc` instead of `ETH_URL`